modules/admin_manual/pages/configuration/user/user_roles.adoc

```diff
@@ -10,10 +10,6 @@
 
 {description}
 
-== User Request for Changing a Role
-
-Users can request a change of their current role under menu:User Name[Settings > Personal > General > Account]. When clicking the button btn:[Request new role], the new role can be entered and an email will be sent to the admin group for further processing.
```

Comment:
This seems to have been added in PR #600 
https://github.com/owncloud/docs-server/pull/600/commits/55530e67c203cb024cfbeac6b4ccf4437dc1f943

That PR description just says:
"Recent changes from core."
So I am not sure where this "feature" came from.
@mmattel do you have any memory of why this was added?

Reply:
Absolutely no idea

Reply:
OK - then the text was provided somehow, but it is not feature that ever existed, AFAIK.
Deleting the text is good.